### DISCUSSION TITLE
Why no search icon in the menu bar?

### DISCUSSION THREAD

**Discussion by alphamodh0** [helloSystem/hello#286] [Q&A]
Why do we use a space at the beginning of the menu bar instead of a search icon (or other icon)?:
![serach barmenu](https://user-images.githubusercontent.com/90866187/145714015-ed562631-9b96-4a0d-a476-860af2bcf9b3.png)

**Comment by probonopd** [comment]
Because it was easier to implement this way.

Improvements would be welcome. But then, most items in the menu are words, and everyone understands "Search" whereas not everyone may understand a tiny icon there. We need the vertical space anyway for typing in the search term, so we can just as well make use of that space by writing out the word "Search".

Please ask such questions that are neither bug reports nor feature request here in https://github.com/helloSystem/hello/discussions, and please do use titles that convey the content.

Thanks!

**Reply by alphamodh0** [reply]
Thanks!, and sorry for my bad.
